Edits/BiomeGlobeDetour.cs

```diff
@@ -21,8 +21,9 @@ public override void UnloadEdits()
 
 	private static void Recipe_FindRecipes(OnRecipe.orig_FindRecipes orig, bool canDelayCheck)
 	{
+		if (Main.dedServ)
+			return;
```

Comment:
Remove this and instead override `IsLoadingEnabled` to return `!Main.dedServ`

Reply:
Why would `Edit` need to override it?

Reply:
So inheriting classes could implement it...?

Reply:
`IsLoadingEnabled ` exists on `ILoadable`